gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValue.java

```diff
@@ -879,4 +892,15 @@ public String toString() {
       getObject().toString() :
       GConstants.NULL_STRING;
   }
+  
+  /**
+   * Throw a runtime exception if this property value can't be represented 
+   * in {@link PropertyValue#MAX_BINARY_LENGTH} bytes.
+   */
+  private void validateBytesLength() {
+    if(rawBytes != null && rawBytes.length> MAX_BINARY_LENGTH){
```

Comment:
Well, there is a [`failsOnError`](https://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-mojo.html#failsOnError) configuration flag that is false by default as well as a missing [`consoleOutput`](https://maven.apache.org/plugins/maven-checkstyle-plugin/checkstyle-mojo.html#consoleOutput). Currently checkstyle is just a friendly reminder for everyone that cares to look into each `target/` folder after the build.